docs/coding-guidelines/javascript.md

```diff
@@ -420,6 +402,18 @@ const myStr = "You're amazing just the way you are.";
 const component = <div className="post"></div>;
 ```
 
+[ES2015 template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are available as an alternative to string concatenation when including variables in a string.
+
+```js
+// Before
+const sumLabel = 'The sum of ' + a + ' and ' + b + ' plus ' + c +
+	' is ' + ( a + b + c );
+```
+```js
+// After
+const sumLabel = `The sum of ${ a } and ${ b } plus ${ c } is ${ a + b + c }`;
+```
```

Comment:
>w00t! I :heart: this!

Thanks! As I was changing the original in "Multi-line Statements" to use `const`, I felt depressed that we'd concatenate a string like that, so I felt it was a nice progression to see how the same example can be improved further using template literals.